lib/Extension/PHPUnit/CodeTransform/Domain/Refactor/GenerateTestMethods.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Phpactor\Extension\PHPUnit\CodeTransform\Domain\Refactor;
+
+use Generator;
+use PHPUnit\Framework\TestCase;
+use Phpactor\CodeBuilder\Domain\Builder\SourceCodeBuilder;
+use Phpactor\CodeBuilder\Domain\Code;
+use Phpactor\CodeBuilder\Domain\Updater;
+use Phpactor\TextDocument\ByteOffset;
+use Phpactor\TextDocument\TextDocument;
+use Phpactor\TextDocument\TextDocumentEdits;
+use Phpactor\TextDocument\TextDocumentUri;
+use Phpactor\TextDocument\WorkspaceEdits;
+use Phpactor\WorseReflection\Core\ClassName;
+use Phpactor\WorseReflection\Core\Exception\NotFound;
+use Phpactor\WorseReflection\Core\Reflection\ReflectionClass;
+use Phpactor\CodeTransform\Domain\SourceCode;
+use Phpactor\WorseReflection\Reflector;
+
+
+class GenerateTestMethods /* implements GenerateMethod */
+{
+    public function __construct(
+        private Reflector $reflector,
+        private Updater $updater,
+    ) {
+    }
+
+    /** @return Generator<string> */
+    public function getGeneratableTestMethods(SourceCode $source): Generator
+    {
+        $classes = $this->reflector->reflectClassesIn($source);
+        if (count($classes->classes()) !== 1) {
+            return;
+        }
+
+        $class = $classes->classes()->first();
+        if (!$class instanceof ReflectionClass) {
+            return;
+        }
+
+        if (!$class->isInstanceOf(ClassName::fromString(TestCase::class))) {
+            dump('Not a unit test');
```

Comment:
@dantleech I don't know why classes extending the `TestCase` class still fall into this case here. Maybe you have an idea.

Reply:
probably rather that the PHPUnit test case does not exist in your test, so the instanceof fails. You could add a `namespace PHPUnit\Framework; class TestCase {}` to the workspace